airflow/kubernetes_deprecated/pod.py

```diff
@@ -0,0 +1,182 @@
+        self.result = result
+        self.labels = labels or {}
+        self.name = name
+        self.ports = ports or []
+        self.volumes = volumes or []
+        self.volume_mounts = volume_mounts or []
+        self.node_selectors = node_selectors or {}
+        self.namespace = namespace
+        self.image_pull_policy = image_pull_policy
+        self.image_pull_secrets = image_pull_secrets
+        self.init_containers = init_containers
+        self.service_account_name = service_account_name
+        self.resources = resources or Resources()
+        self.annotations = annotations or {}
+        self.affinity = affinity or {}
+        self.hostnetwork = hostnetwork or False
+        self.tolerations = tolerations or []
+        self.security_context = security_context
+        self.configmaps = configmaps or []
+        self.pod_runtime_info_envs = pod_runtime_info_envs or []
+        self.dnspolicy = dnspolicy
+
+    def to_v1_kubernetes_pod(self):
+        """
+        Convert to support k8s V1Pod
+
+        @return: k8s.V1Pod
+        """
+        meta = k8s.V1ObjectMeta(
+            labels=self.labels,
+            name=self.name,
+            namespace=self.namespace,
+        )
+        spec = k8s.V1PodSpec(
+            init_containers=self.init_containers,
+            containers=[
+                k8s.V1Container(
+                    image=self.image,
+                    command=self.cmds,
+                    name="base",
+                    env=self.envs,
+                    args=self.args,
+                    image_pull_policy=self.image_pull_policy,
+                )
+            ],
+            image_pull_secrets=self.image_pull_secrets,
+            service_account_name=self.service_account_name,
+            dns_policy=self.dnspolicy,
+            host_network=self.hostnetwork,
+            tolerations=self.tolerations,
+            security_context=self.security_context,
+        )
+
+        pod = k8s.V1Pod(
+            spec=spec,
+            metadata=meta,
+        )
+        for port in self.ports:
+            pod = port.attach_to_pod(pod)
+        for volume in self.volumes:
+            pod = volume.attach_to_pod(pod)
+        for volume_mount in self.volume_mounts:
+            pod = volume_mount.attach_to_pod(pod)
+        for secret in self.secrets:
+            pod = secret.attach_to_pod(pod)
+        for runtime_info in self.pod_runtime_info_envs:
+            pod = runtime_info.attach_to_pod(pod)
+        pod = self.resources.attach_to_pod(pod)
+        return pod
+
+    def as_dict(self):
+        res = self.__dict__
+        res['resources'] = res['resources'].as_dict()
+
+        new_ports = []
+        for port in res['ports']:
+            new_ports.append(port.as_dict())
+        res['ports'] = new_ports
+
+        volume_mounts = []
+        for volume_mount in res['volume_mounts']:
+            volume_mounts.append(volume_mount.as_dict())
+        res['volume_mounts'] = volume_mounts
```

Comment:
```suggestion

        res['volume_mounts'] = [vm.as_dict() for vm in res['volume_mounts']]
```

Reply:
fixed in fixed in https://github.com/apache/airflow/pull/9903/commits/78eb425b4a03bfc478ddbdcafb693746f8c92f7d

Reply:
Awesome :). Wasn't sure how much of that was doable in python 2.7. Much cleaner. Thank you @turbaszek